ci/jenkins/README.md

```diff
@@ -0,0 +1,19 @@
+## Antrea CI: Jenkins
+
+### Reasons for Jenkins
+We have tests as Github Action but Jenkins allows tests running on a cluster of multiple nodes and offers better environment setup options.
+
```

Comment:
I think it would be great to include a list of the 3 Jenkins jobs we have here, with a brief description of each one. I would also include the list of the community tests we run in each job (or if the list is large / dynamic, which filters we use to select the community tests we want to run). I know the information is available in the yaml files, but it takes some time to find and I think it belongs in this README as well.

Reply:
Yes, I agree. Added "List of Jenkins jobs".